firmware/csr_write_benchmark/main.rs

```diff
@@ -15,5 +15,8 @@ fn main() -> ! {
             );
         }
     }
+    unsafe {
+        let _ = miralis_ecall(3);
```

Comment:
See other comment in `abi` crate. We should rather expose a `miralis_print_benchmark` function. The function can do things such as panic on failure as no failure is expected with this ecall.